package.json

```diff
@@ -47,6 +47,7 @@
     "markdown-it-sub": "1.0.0",
     "markdown-it-sup": "1.0.0",
     "moment": "2.18.1",
+    "moment-timezone": "^0.5.14",
```

Comment:
Remove the `^`.
`^` means that the minor versions will be automatically updated, so we prefer to have const version